### PR TITLE
Accelerate label drawing

### PR DESCRIPTION
**Pull request by MatthewKhouzam**
* Don't create text if the state is smaller than 3 pixels
* If the text is the same, don't make a new text, keep the old one
* If the label is empty, don't create a label
* Delete the bitmaps as they are cleared

Signed-off-by: Matthew Khouzam <matthew.khouzam@ericsson.com>

### CODE REVIEW

**Review by bhufmann — APPROVED** [review]
Looks good to me.

**Review by ankusht-work — APPROVED** [review]
LGTM

**Comment by bhufmann** [comment]
Btw, I verified that #59 is still fixed after this change.

File: timeline-chart/src/components/time-graph-state.ts [hunk 1]
```diff
@@ -55,16 +55,19 @@ export class TimeGraphStateComponent extends TimeGraphComponent {
...
-        const textWidth = textObj.getLocalBounds().width;
         const position = {
             x: this._options.position.x + this._options.width < 0 ? this._options.position.x : Math.max(0, this._options.position.x),
             y: this._options.position.y
         }
         const displayWidth = this._options.displayWidth ? this._options.displayWidth : 0;
         const labelText = this._model.label;
         const textPadding = 0.5;
-
+        if (displayWidth < 3){
```
Comment (PatrickTasse) [inline timeline-chart/src/components/time-graph-state.ts:65]: space

File: timeline-chart/src/components/time-graph-state.ts [hunk 2]
```diff
@@ -81,14 +84,24 @@ export class TimeGraphStateComponent extends TimeGraphComponent {
                 displayLabel = partialLabel.concat("...");
             }
         }
-
-        const newTextObj = new PIXI.BitmapText(displayLabel, { fontName: fontName });
-        newTextObj.x = textObjX;
-        newTextObj.y = textObjY;
-        this.displayObject.addChild(newTextObj);
+        if (displayLabel === ""){
```
Comment (PatrickTasse) [inline timeline-chart/src/components/time-graph-state.ts:87]: space

**Review by PatrickTasse — APPROVED** [review]

**Comment by bhufmann** [comment]
Still looks good to me. Thanks for fixing the formatting.

**Pull request merged**